CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,4 @@
 - Fix transport in autoprovision device depending on binding [#235]
-- Allow get list of commands without sending measures (empty payload) [#234]
\ No newline at end of file
+- Allow get list of commands without sending measures (empty payload) [#234]
+Add: allow NGSIv2 for updating active attributes at CB, through configuration (#250)
```

Comment:
I'd suggest to colapse the two new lines into one, as was done for IOTA-JSON, see thread: https://github.com/telefonicaid/iotagent-json/pull/265#discussion_r162923399.

In addition, please follow the same format than other lines, i.e. start each line with `- `

Reply:
Fixed